validator/src/main/java/io/avaje/validation/core/DValidator.java

```diff
@@ -308,6 +317,20 @@ public DValidator build() {
           failfast);
     }
 
+    private boolean hasCustomizations() {
+      return DEFAULT == null
+          || failfast
+          || userInterpolator != null
+          || !otherLocales.isEmpty()
+          || !Locale.getDefault().equals(defaultLocale)
+          || !bundleNames.isEmpty()
+          || !bundles.isEmpty()
+          || !DEFAULT_CLOCK.equals(clockSupplier)
+          || !ZERO.equals(temporalTolerance)
+          || !factories.isEmpty()
```

Comment:
Oh, so this !factories.isEmpty() ... is really only for **_MANUALLY_** added factories to the builder

Reply:
Indeed